### PR TITLE
adding build legal to build task

### PR DESCRIPTION
**Pull request by anthonygarvan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,10 +47,10 @@
     "vinyl-paths": "^1.0.0"
   },
   "scripts": {
-    "build": "gulp build-js && gulp build-sass",
+    "build": "gulp build-js && npm run build-legal && gulp build-sass",
```
Comment (ccostino) [inline package.json]: This seems like a circular/recursive "reference" to me if you will, is it okay to use `npm run ...` to in turn execute another `npm run ...`?

Comment (anthonygarvan) [inline package.json]: yeah npm run just executes a shell command. That could be a gulp task, another npm task, anything.

Comment (anthonygarvan) [inline package.json]: If `npm build` ran `npm build-legal` which ran `npm build`, _that_ would be trouble.

Comment (ccostino) [inline package.json]: 😆 yes, yes it would!  Thanks for explaining!

**Pull request merged**